File: output/llama_4_maverick/codenet/direct_translation/python/java/s838619850.java

import java.util.Scanner;

public class s838619850 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String S = sc.next();
        String T = sc.next();
        int N = S.length();
        int M = T.length();
        int ans = M;
        for (int i = 0; i <= N - M; i++) {
            int cnt = 0;
            for (int j = 0; j < M; j++) {
                if (T.charAt(j) != S.charAt(i + j)) {
                    cnt++;
                }
            }
            ans = Math.min(ans, cnt);
        }
        System.out.println(ans);
    }
}

File: dataset/codenet/Python/Code/s838619850.py
S = input()
T = input()

N = len(S)
M = len(T)
ans = M
for i in range(N-M+1):
    S_2 = S[i:i+M]
    cnt = 0
    for j in range(M):
        if T[j] != S_2[j]:
            cnt += 1
    ans = min(ans, cnt)

print(ans)
